<?xml version="1.0"?>
<launch>
    <arg name="veh_frame_id" default="base_link"/>
    <arg name="lux_frame_id" default="ibeo_lux_frame"/>

    <node pkg="tf" type="static_transform_publisher" name="vehicle_to_$(arg lux_frame_id)" args="0 0 0 0 0 0 $(arg veh_frame_id) $(arg lux_frame_id) 100"/>
        <node pkg="ibeo_lux_driver" type="ros_ibeo_lux" name="ibeo_lux_driver">
            <param name="ip_address" value="192.168.0.1"/>
            <param name="port" type="int" value="12002"/>
            <param name="veh_frame_id" value="$(arg veh_frame_id)"/>
            <param name="sensor_frame_id" value="$(arg lux_frame_id)"/>           
        </node>
</launch>

